hoot-core/src/main/cpp/hoot/core/io/ApiDb.h

```diff
@@ -190,6 +190,18 @@ class ApiDb
    */
   QSqlDatabase getDB() { return _db; }
 
+  static long round(double x);
+  static long round(double x, int precision);
```

Comment:
We have that code sprinkled all over hoot but no good explanation for it (maybe some explanation in the docs).  It basically has to do with the storage precision differences between the hootenanny osm database and the public one.  I'll add some comments.